src/components/NavBar.vue

```diff
@@ -77,7 +77,7 @@
           <div class="navbar-iconWrapper tools-mobile-icon" />
           <span class="nav-mobile-button-text">Tools</span>
         </button>
-        <button class="nav-mobile-button" data-cyId="navbar-editProfile" @click="editProfile">
+        <button class="nav-mobile-button" data-cyId="navbar-openProfile" @click="openProfile">
```

Comment:
Is there a reason you changed the @click method to `openProfile` and added the `openProfile` emit? It seems like it has the same functionality, and if so, I think we should either keep the method/emit `editProfile` or refactor its name, instead of having both.

Reply:
From what i understood changing it would fix the mobile behaviour to be similar to desktop, but i can change i back for now.